.circleci/config.yml

```diff
@@ -27,6 +27,27 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+  smoke_test:
+    circleci_ip_ranges: true
```

Comment:
needed this magic line in there, but can only be for a `job` defined here and can't pass it through to a job defined elsewhere